label.py

```diff
@@ -17,12 +17,15 @@
 
 images_to_show = [i for i in utils.get_images() if i not in labels]
 random.shuffle(images_to_show)
+images_marked = [i for i in utils.get_images() if i in labels]
+images_to_show = images_marked + images_to_show
+image_index = len(images_marked)
 drawing = False
 shifting = False
 changing_shape = False
 box_to_change = {}
 all_boxes = {}
-key_map = {'Escape': 27, 'r': 114, 'Enter': 13, 'Space': 32, 'y': 121, 'n': 110, 'd': 100}
+key_map = {'Escape': 27, 'r': 114, 'Enter': 13, 'Space': 32, 'y': 121, 'left_a': 97, 'right_d': 100}
```

Comment:
Add the new commands to the help.

Reply:
Should I do it in the same PR?

Reply:
Yes, since you're adding the commands here, it makes sense to also update the help here.